main.js

```diff
@@ -15,6 +15,7 @@ const yaml = require('js-yaml');
     const charm_path = core.getInput('charm-path');
     const bundle_path = core.getInput('bundle-path');
     const charmcraft_channel = core.getInput('charmcraft-channel');
+    const upload_image = (core.getInput("upload-image") === "true");
```

Comment:
Fixed. Nice catch

Reply:
You know you're a Python-dev, when you use `contains` instead of `includes` 😆
And as far as I know in Javascript we are supposed to use camel case instead of snake case, is that a problem worth changing?